packages/sidenav/src/Sidenav.ts

```diff
@@ -162,11 +181,17 @@ export class SideNav extends Focusable {
 
     protected override render(): TemplateResult {
         return html`
-            <nav @sidenav-select=${this.handleSelect}>
-                <slot
-                    name="descendant"
-                    @slotchange=${this.handleSlotchange}
-                ></slot>
+            <nav
+                @sidenav-select=${this.handleSelect}
+                aria-label=${ifDefined(this.label)}
+                aria-labelledby=${ifDefined(this.labelledby)}
```

Comment:
This won't point through the shadow boundary and isn't worth applying at this time. Future specs may open the door for this type of label application, at which point we can revisit these sorts of patterns.